tests/integration/servicecatalog/cmd_service_test.go

```diff
@@ -52,6 +52,25 @@ var _ = Describe("odo service command tests", func() {
 		})
 	})
 
+	Context("check search functionality", func() {
+
+		It("should pass with searching for part of a service name", func() {
+
+			// We just use "sql" as some catalogs only have postgresql-persistent and
+			// others dh-postgresql-db. So let's just see if there's "any" postgresql to begin
+			// with
+			output := helper.CmdShouldPass("odo", "catalog", "search", "service", "sql")
```

Comment:
So search fetches the full list of services where some part of name matches.

Reply:
Yes that's the point, (see comments)